### PR TITLE
[16.0][IMP] account_product_fiscal_classification: add sequence

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: account_product_fiscal_classification/models/account_product_fiscal_classification_template.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 class AccountProductFiscalClassificationTemplate(models.Model):
     _name = "account.product.fiscal.classification.template"
     _description = "Fiscal Classification Template"
+    _order = "sequence, id"
```
Comment (legalsylvain) [inline account_product_fiscal_classification/models/account_product_fiscal_classification_template.py]:
```suggestion
    _order = "sequence, name"
```

if two items have the same sequence. (that occures, when updating this module), better to order by name.

File: account_product_fiscal_classification/models/account_product_fiscal_classification.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 class AccountProductFiscalClassification(models.Model):
     _name = "account.product.fiscal.classification"
     _description = "Fiscal Classification"
+    _order = "sequence, id"
```
Comment (legalsylvain) [inline account_product_fiscal_classification/models/account_product_fiscal_classification.py]:
```suggestion
    _order = "sequence, name"
```
same here.

File: account_product_fiscal_classification/models/account_product_fiscal_classification.py [hunk 3]
```diff
@@ -20,6 +21,8 @@ class AccountProductFiscalClassification(models.Model):
         " Classification without removing it.",
     )
 
+    sequence = fields.Integer(sequence=10)
```
Comment (legalsylvain) [inline account_product_fiscal_classification/models/account_product_fiscal_classification.py]:
```suggestion
    sequence = fields.Integer(default=10)
```

typo.

**Review by legalsylvain — CHANGES_REQUESTED** [review]
minor remarks. Otherwise, LGTM.
thanks !

File: account_product_fiscal_classification/models/account_product_fiscal_classification_template.py (hunk 1 above)
Comment (alexis-via) [reply account_product_fiscal_classification/models/account_product_fiscal_classification_template.py]: In the current implementation, there is no "_order", so it is order by id. So "sequence, id" keeps the current order on existing installations.

Comment (legalsylvain) [reply account_product_fiscal_classification/models/account_product_fiscal_classification_template.py]: indeed, but is it what we want ?

Comment (alexis-via) [reply account_product_fiscal_classification/models/account_product_fiscal_classification_template.py]: yes ! That way, we don't change the habits of the users until they decide the exact order they want.

Comment (legalsylvain) [reply account_product_fiscal_classification/models/account_product_fiscal_classification_template.py]: I mean, order by id is wrong, and has been fixed by https://github.com/OCA/account-fiscal-rule/pull/275 monthes ago. 
But this patch has not been back-ported.
I think it should. 
the order by sequence is OK, but please take the opportunity to backport this trivial patch !

thanks !

**Comment by alexis-via** [comment]
On the form view of account.product.fiscal.classification, we have a "notebook" with a single "page"... why ? There are other modules which add tabs to this form view ?

**Comment by legalsylvain** [comment]
> we have a with a single ... why ? 

?

**Comment by alexis-via** [comment]
comment updated. Github dislikes when you write XML tags directly in the text of the comment!

**Comment by alexis-via** [comment]
Ready to merge !

**Comment by dreispt** [comment]
@legalsylvain Can you approve please, so we follow due process?

**Comment by legalsylvain** [comment]
Hi ! 

> On the form view of account.product.fiscal.classification, we have a "notebook" with a single "page"... why ? There are other modules which add tabs to this form view ?

Generally, I add a notebook on form view. It allows to add extra tab in custom / OCA modules.

> @legalsylvain Can you approve please, so we follow due process?

No, my comment (https://github.com/OCA/account-fiscal-rule/pull/379#discussion_r1370332588) is not answered.

But no problem : I backported #275 here (#388), so this PR has just to be rebase, and could be merged.

**Comment by alexis-via** [comment]
rebased

**Pull request closed without merging**